apis/python/test/common.py

```diff
@@ -7,6 +7,14 @@
 import tiledb
 from tiledb.vector_search.storage_formats import storage_formats, STORAGE_VERSION
 
+def get_test_data_path():
+    current_dir = os.path.dirname(__file__)
+    test_data_path = os.path.join(current_dir, "../../../test-data")
```

Comment:
The location from previous PRs has been (at the top level):
```
TileDB-Vector-Search/external/test_data
├── README.md
├── arrays
│   ├── bigann10k
│   ├── fmnistsmall
│   ├── siftsmall
│   └── siftsmall_uint8
└── files
    ├── diskann
    └── siftsmall
```
For broader testing there are arrays with different feature types, different dimensions, and different numbers of vectors.  Attached is the directory tree at the next level down.  
[test_data_tree.txt](https://github.com/TileDB-Inc/TileDB-Vector-Search/files/13953355/test_data_tree.txt)

Per some other discussions, we should probably add a manual mechanism for also including the full sift and full bigann1M arrays for testing and/or benchmarking.  That can be a manual download.  There is also (iirc) a preprocessor macro to enable that in src/include/test/array_defs.h.